### PR TITLE
SNMP Trap Handlers for Ruckus Wireless

### PR DESCRIPTION
**Pull request by h-barnhart**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

A few trap handlers and associated tests for Ruckus Wireless devices. Tested with R720 access points and SmartZone running 3.6.2 firmware.

### CODE REVIEW

**Comment by h-barnhart** [comment]
Noticed PR 10177 adds SNMP Cold Start so I'm going to remove it from this request as it is out of scope.

**Comment by h-barnhart** [comment]
Inspection got stuck, rebased to start recheck process.

File: config/snmptraps.php [hunk 1]
```diff
@@ -59,6 +59,25 @@
         'PowerNet-MIB::outletOff' => \LibreNMS\Snmptrap\Handlers\ApcPduOutletOff::class,
         'PowerNet-MIB::outletOn' => \LibreNMS\Snmptrap\Handlers\ApcPduOutletOn::class,
         'PowerNet-MIB::outletReboot' => \LibreNMS\Snmptrap\Handlers\ApcPduOutletReboot::class,
+        'RUCKUS-EVENT-MIB::ruckusEventAssocTrap' => \LibreNMS\Snmptrap\Handlers\RuckusAssocTrap::class,
+        'RUCKUS-EVENT-MIB::ruckusEventDiassocTrap' => \LibreNMS\Snmptrap\Handlers\RuckusDiassocTrap::class,
+        'RUCKUS-EVENT-MIB::ruckusEventSetErrorTrap' => \LibreNMS\Snmptrap\Handlers\RuckusSetError::class,
+        'RUCKUS-SZ-EVENT-MIB::ruckusSZAPMiscEventTrap' => \LibreNMS\Snmptrap\Handlers\RuckusSzApMiscEvent::class,
+        'RUCKUS-SZ-EVENT-MIB::ruckusSZAPFirmwareUpdatedTrap' => \LibreNMS\Snmptrap\Handlers\RuckusSzApFirmware::class,
```
Comment (murrant) [inline config/snmptraps.php]: Scrutinizer is complaining about some of these classes not existing, did you forget to add them?

**Comment by h-barnhart** [comment]
I must have just copied the snmptraps.php from my test build and forgot I have some traps I have identified for handling but haven't had a chance to collect samples for yet.

**Pull request merged**